components/brave_wallet_ui/components/extension/post-confirmation/index.tsx

```diff
@@ -170,8 +170,20 @@ export function TransactionStatus(props: Props) {
 
   if (tx.txStatus === BraveWallet.TransactionStatus.Error) {
     const providerError = transactionProviderErrorRegistry[tx.id]
+    const errorCode =
+        providerError?.code.providerError ??
+        providerError?.code.zcashProviderError ??
+        providerError?.code.bitcoinProviderError ??
+        providerError?.code.solanaProviderError ??
+        providerError?.code.filecoinProviderError
+
     const errorDetailContent =
-      providerError && `${providerError.code}: ${providerError.message}`
+        providerError && `${errorCode}: ${providerError.message}`
```

Comment:
```suggestion
        providerError && `${errorCode ?? BraveWallet.ProviderError.kUnknown}: ${providerError.message}`
```

Reply:
I moved BraveWallet.ProviderError.kUnknown to the errorCode initialization